tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -865,7 +866,7 @@ def fieldop_where_k_offset(
         a: Field[[IDim, KDim], float64],
         k_index: Field[[KDim], int64],
     ) -> Field[[IDim, KDim], float64]:
-        return where(k_index > 0, a(Koff[-1]), 2.0)
+        return where(k_index > int64(0), a(Koff[-1]), 2.0)
```

Comment:
```suggestion
        return where(k_index > int32(0), a(Koff[-1]), 2.0)
```
this is an index type